tox/venv.py

```diff
@@ -149,9 +150,27 @@ def is_allowed_external(self, p):
                     return True
         return False
 
+    def _real_python3(self, python):
+        """ use real_prefix to determine if we're running inside a virtualenv,
+            and if so, use it as the base path to determine the real python
+            executable path.
+        """
+        args = [str(python), '-c', 'import sys; print(sys.real_prefix)']
```

Comment:
`sys.real_prefix` is only set by virtualenv. If we are not presently in a virtualenv, this attr doesn't exist and the command will fail.